scripts/postinstall.sh

```diff
@@ -11,7 +11,7 @@ for x in $(seq 0 2) ; do
 done
 
 # Remove install ssh key
-rm -rf /root/.ssh /root/.packer_http
+rm -rf /nixos/.ssh /nixos/.packer_http
```

Comment:
No reason to write `.packer_http`, so this can be omitted too, right?

Reply:
Yes, i remove it.